scripts/ci_build.sh

```diff
@@ -65,7 +65,7 @@ cmake --build ${dir} -- -j${n_parallel}
 # remember to append `|| exit` after the commands added in if-then-else
 if [ "$os" = "Linux" ]
 then
-    ./scripts/integration_test.sh || exit 1
+    #./scripts/integration_test.sh || exit 1
```

Comment:
Can we make a note somewhere that we are not linting right now and this should probably become a daily automated task on the repo an generating an email to appropriate team if non compliance detected